metricbeat/module/mongodb/currentops/currentops.go

```diff
@@ -3,14 +3,19 @@ package currentops
 import (
 	"fmt"
 
+	"github.com/elastic/beats/libbeat/common"
 	s "github.com/elastic/beats/libbeat/common/schema"
 	"github.com/elastic/beats/libbeat/logp"
 	"github.com/elastic/beats/metricbeat/mb"
 	"github.com/elastic/beats/metricbeat/module/mongodb"
+	"github.com/go-redis/redis"
 	"github.com/pkg/errors"
 	"gopkg.in/mgo.v2/bson"
 )
 
+// Connection info for redis
+const redisConnectionString = "localhost:6379"
```

Comment:
localhost?, will this run on redis Host?

Reply:
No bro. Will use environment variables for redis connection. 
This is just mock.